lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -42,6 +42,7 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
     this._customValidityErrorMessage = "";
 
     this._labels = null;
+    this._selectedOption = null;
```

Comment:
This doesn't seem like the right name, since sometimes `this._selectedOption._selectedness` will be false. And it would probably benefit from an explanatory comment. Maybe something like `_lastSelectedOptionForFastPath`?

Reply:
Renamed to `_lastSelectedOptionForFastPath` and added a comment explaining reason for field.